### PR TITLE
support treesitter

### LINKED ISSUE #557: support Compatible with treesitter
After checking the source code, I found that after enabling tresitter html
  let type = synIDattr(synID(max([pos[1], 1]), max([pos[2], 1]), 1), 'name')
The return value of

### PR DESCRIPTION
**Pull request by kola-web**
Fixes #557

### CODE REVIEW

File: autoload/emmet.vim [hunk 1]
```diff
@@ -379,8 +379,12 @@ function! emmet#getFileType(...) abort
     endif
   endif 
 
-  let pos = emmet#util#getcurpos()
-  let type = synIDattr(synID(max([pos[1], 1]), max([pos[2], 1]), 1), 'name')
+  if exists('g:loaded_nvim_treesitter') == 1
```
Comment (mattn) [inline autoload/emmet.vim:382]:
```suggestion
  if get(g:, 'loaded_nvim_treesitter', 0)
```

**Comment by mattn** [comment]
please rename lua filename like emmet_utils.lua

**Comment by kola-web** [comment]
ok

**Comment by kola-web** [comment]
Revised

**Comment by mattn** [comment]
Thank you

**Pull request merged**